.pipelines/scripts/run-e2e.sh

```diff
@@ -83,5 +83,14 @@ kubectl wait --for=condition=Ready node --all --timeout=5m
 kubectl get node -owide
 
 echo "Running e2e"
+
+# TODO: We should do it in autoscaling-multipool.json
+if [[ "${CLUSTER_TYPE}" == "autoscaling-multipool" ]]; then
+  az aks update --resource-group "${RESOURCE_GROUP}" --name "${CLUSTER_NAME}" --cluster-autoscaler-profile balance-similar-node-groups=true 
+fi
+
 export E2E_ON_AKS_CLUSTER=true
-make test-ccm-e2e
+if [[ "${CLUSTER_TYPE}" =~ "autoscaling" ]]; then
+  export LABEL_FILTER="Feature:Autoscaling"
+fi
+make test-ccm-e2e
```

Comment:
Please add an empty line below.

Reply:
> Please add an empty line below.

Updated. And also change the REPO_ROOT to help "make test-ccm-e2e" find KUBECONFIG file, otherwise error like "no configuration" will come out. PTAL.